morph/Visual.h

```diff
@@ -255,6 +258,30 @@ namespace morph {
         // A callback friendly wrapper
         static void set_context (morph::Visual<glver>* _v) { _v->setContext(); };
 
+        //! Lock the context mutext to prevent locking across multiple. Then sets the context.
+        void lockContext() { 
+            context_mutex.lock(); 
+            setContext();
+        }
+        
+        //! Attempt to lock the context mutext returns if locking was successful. 
+        //! If success sets the context.
+        bool tryLockContext() { 
+          if(context_mutex.try_lock()){
+            set_context();
```

Comment:
Shouldn't this be setContext()?

Reply:
yeah not sure how this got through but it did compile. updated to match the call.